.github/workflows/ci.yml

```diff
@@ -32,3 +31,23 @@ jobs:
         bundler-cache: true # runs 'bundle install' and caches installed gems automatically
     - name: Run tests
       run: bundle exec rake
+
+  lint:
```

Comment:
For performance sake, can we add rubocop just as a step before the tests? It would spare a full spin up of a machine just for running rubocop.

Reply:
Yes that makes sense.